plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/delete/DeleteProjectPresenter.java

```diff
@@ -141,6 +146,7 @@ public void apply(PromiseError arg) throws OperationException {
         return new Operation<PromiseError>() {
             @Override
             public void apply(PromiseError promiseError) throws OperationException {
+                loader.hide();
```

Comment:
Loader can be hided twice https://github.com/codenvy/che-plugins/pull/417/files#diff-25f8f55b1c8aa28d2c110276ec634a95R120 and https://github.com/codenvy/che-plugins/pull/417/files#diff-25f8f55b1c8aa28d2c110276ec634a95R135 What does happened if call hide method in loader twice?